core/metrics-core-service/src/test/java/org/hawkular/metrics/core/service/metrics/BaseMetricsITest.java

```diff
@@ -93,7 +93,7 @@ public void initClass() {
     @BeforeMethod(alwaysRun = true)
     public void initMethod() {
         session.execute("TRUNCATE tenants");
-        session.execute("TRUNCATE data");
+//        session.execute("TRUNCATE data");
```

Comment:
Why is this commented out? If not needed, can you please remove and/or replace with a note for why data is not truncated?